src/main/java/com/arpnetworking/metrics/jvm/collectors/NonHeapMemoryMetricsCollector.java

```diff
@@ -16,21 +16,20 @@
 package com.arpnetworking.metrics.jvm.collectors;
 
 import com.arpnetworking.metrics.Metrics;
-import com.arpnetworking.metrics.Units;
 import com.arpnetworking.metrics.jvm.ManagementFactory;
 
 import java.lang.management.MemoryPoolMXBean;
-import java.lang.management.MemoryUsage;
+import java.lang.management.MemoryType;
 import java.util.List;
 
 /**
- * Collector class for JVM memory usage metrics for each memory pool. Uses the Java Management API to get the metrics
- * data.
+ * Collector class for JVM memory usage metrics for each non-heap memory pool. Uses the Java Management API to get the
+ * metrics data.
  *
  * @author Deepika Misra (deepika at groupon dot com)
 */
 // CHECKSTYLE.OFF: FinalClass - Allow clients to inherit from this.
-public class NonHeapMemoryMetricsCollector implements JvmMetricsCollector {
+public class NonHeapMemoryMetricsCollector extends PoolMemoryMetricsCollector {
```

Comment:
How does this now relate to HeapMemoryMetricsCollector.java?

Reply:
They're completely different.  As discussed, this is a change of behavior to be in line with what you would expect from the name of the class.  They use different MX Beans and essentially do different things.